portafly/src/components/shared/data-list/reducers/DataReducer.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  ActionHandlers,
+  Action,
+  createReducer
+} from 'utils'
+
+export type DataState = { id: number }[]
```

Comment:
We could add more fields when we understand better the needs for all the data sets from our API

Reply:
Maybe it's better to use an `interface` for this 🤔 so that we can extend it.